### PR TITLE
final html-css training cards

### PR DESCRIPTION
**Pull request by MicaelaBalza**
### Summary
- Finished all final training cards

### Screenshot

Transitions:
Transitions

Grid Refactor:
grid-mobile
grid-tablet

Animation:
animation

### Trello Card
https://trello.com/c/1FFpSVgc/34-responsive
https://trello.com/c/3Pb7m8xu/35-transitions
https://trello.com/c/g4aL1cQP/12-redise%C3%B1o-de-la-p%C3%A1gina
https://trello.com/c/7yItmLq8/36-animations

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -1,82 +1,164 @@
...
+    padding: 32px 32px 56px;
+  }
+
+  .title {
+    font-size: 32px;
+  }
+
+  .genre {
+    font-size: 24px;
+    margin-top: -44px;
```
Comment (SaraIsabelSantacruz) [inline scss/detail.scss]: don't use negative margins

File: scss/detail.scss [hunk 2]
```diff
@@ -1,82 +1,164 @@
...
   }
 }
+
+.green-underline {
+  position: relative;
+
+  &::after {
+    content: "";
+    background-color: $earls-green;
+    bottom: -16px;
```
Comment (SaraIsabelSantacruz) [inline scss/detail.scss]: better use a positive value for top

File: scss/navbar.scss [hunk 3]
```diff
@@ -5,6 +5,7 @@
   position: sticky;
   top: 0;
   width: 100%;
+  z-index: 1000;
```
Comment (SaraIsabelSantacruz) [inline scss/navbar.scss]: why 1000?? 😅

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**